docs/en/UI/AspNetCore/Tag-Helpers/Popovers.md

```diff
@@ -0,0 +1,42 @@
+# Popovers
+
+## Introduction
+
+`abp-popover` is the abp tag for popover messages.
+
+Basic usage:
+
+````xml
+<abp-button abp-popover="Hi, i'm popover content!">
+    Popover Default
+</abp-button>
+````
+
+
+
+## Demo
+
+See the [popovers demo page](https://bootstrap-taghelpers.abp.io/Components/Popovers) to see it in action.
+
+## Attributes
+
+### disabled
+
+A value indicates if the element should be disabled for interaction. If this value is set to `true`, `dismissable` attribute will be ignored. Should be one of the following values:
+
+* `false` (default value)
+* `true`
+
+### dismissable
+
+A value indicates to dismiss the popovers on the user's next click of a different element than the toggle element. If this value is set to `true`, `hoverable` attribute will be ignored. Should be one of the following values:
```

Comment:
>  If this value is set to `true`, `hoverable` attribute will be ignored.

This is outdated. See https://github.com/abpframework/abp/pull/3282#discussion_r396247355

Reply:
Absolutely.